app/assets/javascripts/application.js

```diff
@@ -16,4 +16,10 @@
 //= require turbolinks
 //= require_tree .
 
-$(function(){ $(document).foundation(); });
```

Comment:
@liammccartney Nope.  Don't put code in here.  How about an `initialize.js` file.  The closure pattern here (known as an IIFE) is nice though.